tests/hawtio-test-suite/src/main/java/io/hawt/tests/features/pageobjects/fragments/camel/tabs/common/CamelChart.java

```diff
@@ -87,7 +90,7 @@ public void checkSpecificAttributeIsNotDisplayed(String attributeName) {
      * @param value     string value of tested attribute
      */
     public void checkStringAttributeValue(String attribute, String value) {
-        Assertions.assertEquals(getValue(attribute), value);
+        Assertions.assertEquals(value, getValue(attribute));
```

Comment:
Maybe you can rename the method args to make it more visible which one is expected and which one is the value to test.

Reply:
Sure, I will rename it. Thanks